aws/table_aws_accessanalyzer_finding.go

```diff
@@ -0,0 +1,281 @@
+package aws
+
+import (
+	"context"
+	// "strings"
+
+	"github.com/aws/aws-sdk-go-v2/service/accessanalyzer"
+	"github.com/aws/aws-sdk-go-v2/service/accessanalyzer/types"
+
+	accessanalyzerv1 "github.com/aws/aws-sdk-go/service/accessanalyzer"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+type accessanalyzerFindingInfo = struct {
+	Finding types.Finding
+	AccessAnalyzerArn     string
+}
+
+func tableAwsAccessAnalyzerFinding(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_accessanalyzer_finding",
+		Description: "AWS Access Analyzer Finding",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"id", "access_analyzer_arn"}),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ResourceNotFoundException", "ValidationException", "InvalidParameter"}),
+			},
+			Hydrate: getAccessAnalyzerFindings,
+			Tags:    map[string]string{"service": "access-analyzer", "action": "GetFinding"},
+		},
+		List: &plugin.ListConfig{
+			ParentHydrate: listAccessAnalyzers,
+			Hydrate:       listAccessAnalyzersFindings,
+			Tags:          map[string]string{"service": "access-analyzer", "action": "ListFindings"},
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "access_analyzer_arn",
+					Require: plugin.Optional,
```

Comment:
Can't we add more optional quals for this table?
As per the API doc, there is an input parameter [Filter](https://docs.aws.amazon.com/access-analyzer/latest/APIReference/API_ListFindings.html#accessanalyzer-ListFindings-request-filter) for the API `ListFindings`. Not sure though what are the attributes supported by the filter parameter.

Reply:
I don't think so we can add but I am not sure this is the  [Criterion structure](https://pkg.go.dev/github.com/aws/aws-sdk-go/service/accessanalyzer#Criterion)